### ISSUE TITLE
Review and discuss `usersecrets` sub-generator under `.csproj`

### ISSUE THREAD

**Issue by peterblazejewicz** [OmniSharp/generator-aspnet#865]
the `usersecrets` sub-generator cannot be directly ported to `.csproj` as the container file format is being chagned (JSON > XML).
The expected output including new user secrets reference:
```xml
  <PropertyGroup>
    <TargetFramework Condition="'$(Framework)' != '1.1'">netcoreapp1.0</TargetFramework>
    <TargetFramework Condition="'$(Framework)' == '1.1'">netcoreapp1.1</TargetFramework>
    <UserSecretsId Condition="'$(IndividualAuth)' == 'True'">aspnet-Company.WebApplication1-53bc9b9d-9d6a-45d4-8429-2a2761773502</UserSecretsId>
    <MvcRazorCompileOnPublish Condition="'$(Framework)' == '1.1'">false</MvcRazorCompileOnPublish>
  </PropertyGroup>
```
https://github.com/dotnet/templating/blob/b96ec9c36f8c1accf6dcc608da66c768ac17ab5f/template_feed/Microsoft.DotNet.Web.ProjectTemplates/content/StarterWeb-CSharp/Company.WebApplication1.csproj#L3-L8

**Comment by peterblazejewicz** [comment]
The user secrets are now always part of content if project supporting authentication is created (see above). Also:
https://github.com/dotnet/templating/blob/master/template_feed/Microsoft.DotNet.Web.ProjectTemplates/content/StarterWeb-CSharp/Startup.cs#L35

For this reason lets start with removing user-secrets subgenerator completely and return to topic later (I've been using user-secrets in other type of projects - for example when keys for 3rd party service like Auth0 were required to be stored and used in app)

**Comment by peterblazejewicz** [comment]
closed via: 3a78d77
